python/foglamp/services/core/api/browser.py

```diff
@@ -333,6 +343,10 @@ def validate_limit_skip(request, _dict):
     _limit_skip_payload = validate_limit_skip(request, d)
     d.update(_limit_skip_payload)
 
+    # Add ORDER BY timestamp DESC
+    _sort_payload = PayloadBuilder(_limit_skip_payload).ORDER_BY(["timestamp", "desc"]).chain_payload()
+    d.update(_sort_payload)
+
     payload = json.dumps(d)
```

Comment:
@pintomax FYI, Instead of above code snippet we could have been use `payload = PayloadBuilder(d).ORDER_BY(["timestamp", "desc"]).payload()`